PublicMapping/districtbuilder
# Upgrade Terraform AWS provider to 3.x

## Overview

Upgrade the Terraform AWS provider to version 3.x. Also, update to a version of the VPC and ACM module with support for this version of the Terraform AWS provider.

### Checklist


### Notes

Warnings related to the `districtbuilder_state_count` variable may surface during Terraform interactions until https://github.com/PublicMapping/districtbuilder/pull/309 is merged and rebased.

## Testing Instructions

Ensure that a Terraform plan completes successfully without any resource changes:

```console
$ docker-compose -f docker-compose.ci.yml run --rm terraform
bash-5.0# GIT_COMMIT="fff8272" ./scripts/infra plan

. . .

No changes. Infrastructure is up-to-date.

This means that Terraform did not detect any differences between your
configuration and real physical resources that exist. As a result, no
actions need to be performed.
```
